src/WebUI/Pages/Shared/_Layout.cshtml

```diff
@@ -24,7 +24,9 @@
                 }
                 else
                 {
-                    <a class="header--item" asp-page="/Logout">Sign Out</a>
+                    <form method="post">
+                        <button class="header--item" asp-page="/Logout">Sign Out</button>
```

Comment:
You want to move the `asp-page` to the form in order to make that the target action.

Reply:
It worked either way but putting a type submit and the link on the form looks cleaner so OK.